README.md

```diff
@@ -1,208 +1,262 @@
-To do so follow these steps to get registered for Google cloud messaging and enter the details into the Mendix screens.
+Open your application in the browser and login as administrator.
 
-### Step 1 - Login to developers console
-Open up the google [developers console] (https://console.developers.google.com) and login with your google id.
-<img src="assets/images/gcm-step1.PNG"/>
+### Step 3 - Open the "Device Overview" page
 
-### Step 2 - Create project
-Click new project and fill in the project name and project ID for your application. Then click create. Once created you will see a project number at the top of the screen next to the project ID. Take note of this ID because you will need it later on for our sender ID. 
-<img src="assets/images/gcm-step2.PNG"/>
+Navigate to the `Device Overview` page. Here you should see that one device has already been registered, which is the device that you use to login to your hybrid mobile application. Continue by selecting the device and press the `New Message` button.
 
-### Step 3 - Enable an API
-Once created click on the button enable an API. 
-<img src="assets/images/gcm-step3.PNG"/>
+<img src="assets/images/testing/device overview.JPG"/>
 
-### Step 4 - Enable Google Cloud Messaging
-Locate the Google Cloud Messaging for Android and click the off button to turn on the API.
-<img src="assets/images/gcm-step4.PNG"/>
+### Step 4 - Send a message
 
-### Step 5 - Create server keys
-Click on the menu option credentials, located on the left hand side under the APIs & Auth section. Then click on the button under the public API Access that says create new key. When the popup appears press the Server key button and then press create on the next screen. Take note of the API Key because you will need this when we set up the push notifcation in Mendix.
-<img src="assets/images/gcm-step5.PNG"/>
+Fill in the title and the message on the form and press `Send immediate` button. You should see a new notification coming in your device. By default, if your hybrid mobile app is currently active, the message will be displayed in the app. Otherwise, it will be shown as a standard push notification.
 
-### Step 6 - Setup Mendix app
-Open up your application in Mendix and login as an Admin, so that you can see the menu option google admin.
-Enter the project ID into the sender ID field and the API key into the API field. 
-<img src="assets/images/gcm-step6.PNG"/>
+<img src="assets/images/testing/send message.JPG"/>
 
-Once entered tick the checkbox enabled and press restart. From now on your application will always start the gcm push notification system for you.
+## APIs
 
-For more information on setting up your google API then please refer to this article: (Google API Setup) [http://developer.android.com/google/gcm/gs.html]
+This is a list of microflows that can be called from within your application. They are located in the `_USE ME/Microflows` folder.
```

Comment:
'that can be called by your application to send push notifications.' or something like that maybe?